mysql-test/suite/rpl/r/rpl_init_slave_errors.result

```diff
@@ -15,8 +15,10 @@ connection slave;
 reset slave;
 SET GLOBAL init_slave= "garbage";
 start slave;
-include/wait_for_slave_sql_error.inc [errno=1064]
-Last_SQL_Error = 'Slave SQL thread aborted. Can't execute init_slave query'
+include/wait_for_slave_sql_error.inc [errno=4210]
+Last_SQL_Error = 'Slave SQL thread aborted. Can't execute init_slave query, due to error code 1064: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'garbage' at line 1'
```

Comment:
🆙 for including the underlying error’s message!

This is reminiscent of some programming languages where one exception can annotate another as its `cause`.
In contrast, the error code system is limited to a single layer.